x-pack/plugins/enterprise_search/tsconfig.json

```diff
@@ -61,7 +60,7 @@
     "@kbn/global-search-plugin",
     "@kbn/logs-shared-plugin",
     "@kbn/share-plugin",
-    "@kbn/core-saved-objects-migration-server-internal",
```

Comment:
this related?

Reply:
That must be a merge fuckup, let me fix that.